src/v2/Apps/Artwork/Components/TrustSignals/BuyerGuarantee.tsx

```diff
@@ -0,0 +1,47 @@
+import { CheckCircleIcon, Link } from "@artsy/palette"
+import React from "react"
+import { BuyerGuarantee_artwork } from "v2/__generated__/BuyerGuarantee_artwork.graphql"
+
+import { createFragmentContainer, graphql } from "react-relay"
+import { TrustSignal } from "./TrustSignal"
+
+interface Props {
+  artwork: BuyerGuarantee_artwork
+}
+
+const BuyerGuarantee: React.FC<Props> = props => {
+  const { artwork } = props
+  return (
+    (artwork.is_acquireable || artwork.is_offerable) && (
```

Comment:
I always need to look up again and make sure acquireable means Buy Now-able. 😅

Reply:
haha, yes I new the `SecurePayment` component works the same so used the same [logic](https://github.com/artsy/force/blob/master/src/v2/Apps/Artwork/Components/TrustSignals/SecurePayment.tsx#L18)